bumper.js

```diff
@@ -43,21 +43,23 @@ module.exports = function bumper(config = {}) {
 
                 //console.log(commits);
                 commits.forEach(commit => {
-                    addBangNotes(commit); //add notes if breaking change or with "!"
+                    if(commit.merge === null){
```

Comment:
Looks ok according to [parser source](https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-commits-parser/lib/parser.js#L117); however the whitespace should follow our code style.
```suggestion
                    if (commit.merge === null) {
```

Reply:
I guess I should setup prettier for that repo too